Doc change - Tags editor in AB has usability problem [PAB-4885] (#3621)
SoftwareAG/c8y-docs: content/streaming-analytics/analytics-builder-bundle/using-the-model-editor.md

1.  In the model editor, click **Model settings** which is shown at the left of the toolbar.
2.  In the resulting **Edit model** dialog box, specify a new unique name for the model, change the description, and/or change the tags.

    To add a tag, you can either:

>      - Type its name and press the Enter key. The tag is then shown in a colored rectangle.

Reviewer: ```suggestion
     - Type its name and press Enter. The tag is then shown in a colored rectangle.
```

Author: Updated.

>      - Type its name and submit the dialog without pressing Enter. Any pending text in the tag field will automatically be added as a tag before submission.

Reviewer: ```suggestion
     - Type its name and submit the dialog without pressing Enter. Any pending text in the **Tag** field will automatically be added as a tag before submission.
```

Reviewer: If this is a field with a label (Tag) it must be highlighted in bold.

Author: Updated.

>     To remove a tag, click on the X that is shown in the rectangle.

Reviewer: ```suggestion
    To remove a tag, click on the delete icon (X) that is shown in the rectangle.
```

Author: Updated.

>     The dialog prevents you from entering duplicate tags for a model; if you enter such a tag name, the duplicate tag is not added and the original tag blinks one time.

Reviewer: ```suggestion
    The dialog prevents you from entering duplicate tags for a model. If you enter such a tag name, the duplicate tag is not added, and the original tag blinks once.
```

Author: Updated.
